src/lib/pickles/test/optional_custom_gates/pickles_test_optional_custom_gates.ml

```diff
@@ -281,6 +281,31 @@ let main_fixed_lookup_tables_multiple_tables_multiple_lookups () =
            ; w6 = exists Field.typ ~compute:(fun () -> v3)
            } ) )
 
+let main_runtime_table_cfg () =
+  let table_ids = Array.init 5 ~f:(fun i -> Int32.of_int_exn i) in
+  let size = 10 in
+  let first_column = Array.init size ~f:Field.Constant.of_int in
```

Comment:
3 mini points here:
- pick one style indirect `let size = ... in Array.init size` or the direct `Array.init <size>` (indirect is ok)
- why 5 or 10? could they be optional labeled parameters?
-  prefer`Int32.of_int_exn` to`(fun i -> Int32.of_int_exn i)